8_kyu/welcome.rb

```diff
@@ -0,0 +1,25 @@
+# https://www.codewars.com/kata/577ff15ad648a14b780000e7
+
+def greet(language)
+  data = {
+    'english'=>'Welcome',
+    'czech'=>'Vitejte',
+    'danish'=>'Velkomst',
+    'dutch'=>'Welkom',
+    'estonian'=>'Tere tulemast',
+    'finnish'=>'Tervetuloa',
+    'flemish'=>'Welgekomen',
+    'french'=>'Bienvenue',
+    'german'=>'Willkommen',
+    'irish'=>'Failte',
+    'italian'=>'Benvenuto',
+    'latvian'=>'Gaidits',
+    'lithuanian'=>'Laukiamas',
+    'polish'=>'Witamy',
+    'spanish'=>'Bienvenido',
+    'swedish'=>'Valkommen',
+    'welsh'=>'Croeso'
+  }
+  return data[language] || data['english']
```

Comment:
For what return here? I Ruby latest statement is always return context. Also you can define it using Hash default value feature. https://ruby-doc.org/core-3.1.2/Hash.html#method-c-new

```ruby
def greet(language)
  Hash.new('Welcome').merge(
    'czech'=>'Vitejte',
    'danish'=>'Velkomst',
    'dutch'=>'Welkom',
    'estonian'=>'Tere tulemast',
    'finnish'=>'Tervetuloa',
    'flemish'=>'Welgekomen',
    'french'=>'Bienvenue',
    'german'=>'Willkommen',
    'irish'=>'Failte',
    'italian'=>'Benvenuto',
    'latvian'=>'Gaidits',
    'lithuanian'=>'Laukiamas',
    'polish'=>'Witamy',
    'spanish'=>'Bienvenido',
    'swedish'=>'Valkommen',
    'welsh'=>'Croeso'
  )
end
```